packages/react-dom/src/__tests__/ReactDOMFiberAsync-test.js

```diff
@@ -730,7 +730,7 @@ describe('ReactDOMFiberAsync', () => {
     });
 
     // @gate enableFrameEndScheduling || !allowConcurrentByDefault
-    it('When allowConcurrentByDefault is enabled, unknown updates should not be time sliced', () => {
+    it.skip('When allowConcurrentByDefault is enabled, unknown updates should not be time sliced', () => {
```

Comment:
These tests are currently failing. Should default updates and unknown updates be time sliced?